nums/core/utils.py

```diff
@@ -11,4 +11,4 @@
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
-# limitations under the License.
+# limitations under the License.
```

Comment:
Why was this file modified?

Reply:
Actually, remember that I had added the find_slices function incorrectly over here..so I deleted that.